### ISSUE TITLE
Kit become non-responsive sometimes after mac sleep.

### ISSUE THREAD

**Issue by re4388** [johnlindquist/kit#1266]
It sometimes happened when I close my mac lid.
I didn’t change the default setting, so I guess mac goes into some kind of mac sleep mode when I close the lid.
When I open my lid and begin to use kit, that will be hit the trigger key (in my case, right_cmd+space, not sure if this is relevant thought)

kit become non-responsive
I need to force kill it and restart.

---
my spec
![image](https://github.com/johnlindquist/kit/assets/5414334/7ee2e6c4-5bf6-4905-b3ba-822d0821cab5)

kit version
![image](https://github.com/johnlindquist/kit/assets/5414334/e1d17dd9-9a5f-47cc-842a-5d85efb792c2)

log when this happen
![image](https://github.com/johnlindquist/kit/assets/5414334/b64f0639-507d-41da-ac39-8de1b3e83600)

log when everything goes well
![image](https://github.com/johnlindquist/kit/assets/5414334/b2075f93-caa5-4f53-9fb9-b8aa304b8d35)

**Comment by johnlindquist** [comment]
@re4388 looks like it's freezing when trying to start up the keyboard watching library. 🤔

Do you have other software on your machine that watches the keyboard?

Next time it happens, can you try clicking on the tray and trying the:

Watchers -> Restart Watcher

To see if that "unfreezes"?

**Comment by re4388** [comment]
When this happen, I am unable to click the tray.
The app is hanging and shows the mac loading symbol (like below, just to make it clear XD ) when I move over my cross to the tray.
![image](https://github.com/johnlindquist/kit/assets/5414334/cf7a3e52-5b93-4840-a8c3-3deb6182d12f)

---

I also captured another log when this happened:
![image](https://github.com/johnlindquist/kit/assets/5414334/59f0c851-6138-4728-ad50-0027f5cbc9f2)

So maybe it is related to watcher.

--

I think when I first use Kit, couple month ago, I don't have this issue. 
And I kind of remember the older version is not enabled the `watcher` by default.
I remember I had tried to "stop" the watcher but this 'sometimes awake Kit hanging` situation continues.
Maybe the watcher code will still run when mac awake and cause the issue?
Is there hacky way I can just disable the watcher/lib so I can experiment if this is the root cause?

--

I do have some app might also watch keyboard.
aText -> I can try to disable it and see if anything goes well.
karabiner-element -> cannot disable it, kind of remapping many important key on it XD
keyboard maestro -> also cannot disable it since many necessary remapping.
